integration-tests/utils/integration-utils.js

```diff
@@ -33,16 +33,29 @@ const findInGrid = (page, textContains, options) => {
 }
 
 const clickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a | //button | //input[@type="checkbox"] | //*[@role="button" or @role="checkbox"] | //*[@role="link"])${isDescendant ? '//*' : ''}`
+  const base = `(//a | //button | //input[@type="checkbox"] | //*[@role="button" or @role="checkbox"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])${isDescendant ? '//*' : ''}`
```

Comment:
The `or @role="checkbox"` here is causing the test to fail to click the "Preview data" button and changing it to `| [@role="checkbox"]` doesn't make a difference 🤔

Reply:
It turns out the test does not reach this page because this function clicks the checkbox element in the list of datasets instead of the dataset title!